apps/condo/domains/marketplace/access/MarketItemFile.js

```diff
@@ -38,15 +38,24 @@ async function canReadMarketItemFiles ({ authentication: { item: user }, origina
     }
 
     return {
-        marketItem: {
-            organization: {
-                OR: [
-                    queryOrganizationEmployeeFor(user.id, 'canReadMarketItems'),
-                    queryOrganizationEmployeeFromRelatedOrganizationFor(user.id, 'canReadMarketItems'),
+        OR: [
+            {
+                AND: [
+                    {
+                        marketItem: {
+                            organization: {
+                                OR: [
+                                    queryOrganizationEmployeeFor(user.id, 'canReadMarketItems'),
+                                    queryOrganizationEmployeeFromRelatedOrganizationFor(user.id, 'canReadMarketItems'),
+                                ],
+                                deletedAt: null,
+                            },
+                        },
+                    },
                 ],
-                deletedAt: null,
             },
-        },
+            { createdBy: { id: user.id } },
```

Comment:
It looks like the user who created a model and then leave organization will still has read/write access to model. Is it ok?